docs/guides/context.md

```diff
@@ -67,17 +67,58 @@ type Post {
 
 In this case, `value.userId` is a way to get the `userId` information from the "parent" context of the `Post` type. Essentially, it's extracting a list or "array" of `userId` fields from multiple `Post` types. Think of `value` as a container that holds the results of a post query, with `userId` being the specific key you want to fetch from that container.
 
-### env
+### Environment Variables
 
-This represents global environment variables for the server. This is set once when the server starts.
+Environment variables allow you to configure different aspects of your Tailcall server without needing to hardcode values. Some common uses:
 
-```graphql showLineNumbers
-type Query {
-  users: [User]! @http(baseUrl: "{{env.API_ENDPOINT}}", path: "/users")
+Setting the base URL for API requests:
+```type Query {
+  users: [User]! @http(baseUrl: "{{env.API_ENDPOINT}}", path: "/users") 
 }
 ```
 
-In the above example, `env.API_ENDPOINT` refers to an environment variable called API_ENDPOINT, which should be defined in your server settings.
+Toggling features on/off:
+```type Query {
```

Comment:
Use the following syntax.

```
```graphql
type Query
```

```

Currently it's 

```
```type Query
```

```